client/python/test/test_client.py

```diff
@@ -64,12 +67,24 @@ def test_numerical_to_label_space(self):
         ):
             client.predict("male", 77, 10, api_name="predict")
 
+    @pytest.mark.flaky
+    def test_numerical_to_label_space_v4(self):
```

Comment:
I ported over the spaces to use v4 gradio so we can have coverage before release